config/settings/idb.q

```diff
@@ -0,0 +1,13 @@
+// Bespoke IDB config
+
+// Server connection details
+\d .servers
+CONNECTIONS:()                                                                              // list of connections to make at start up
+STARTUP:1b                                                                                  // create connections
+
+\d .proc
+loadprocesscode:1b                                                                          // whether to load the process specific code defined at ${KDBCODE}/{process type}
+
+\d .idb
+savedir:hsym`$getenv[`TORQHOME],"/wdbhdb"                                                   // location of the wdb data
+hdbdir:hsym`$getenv[`TORQHOME],"/hdb"                                                       // location of the wdb data
```

Comment:
I think these should be relative to TORQDATA rather than TORQHOME
I think what you have aligns with the original version of TorQ, but setenv has grown over time I think and we now have KDBHDB/KDBWDB which are relative to TORQDATA. We've left the original config as it was for backward compatability. 

For consistency we could make KDBIDB, or just make it in here relative to TORQDATA. I leave it up to you.

Reply:
You are right, added KDBHDB/KDBWDB.